widget/ui/src/components/Alert/Alert.types.ts

```diff
@@ -8,5 +9,8 @@ export interface PropTypes {
   footer?: ReactNode;
   action?: ReactNode;
   variant?: 'alarm' | 'regular';
-  containerStyles?: CSSProperties;
+  containerStyles?: { [key: string]: string };
```

Comment:
I think `css` from stiches has its own types. Could you please check?

Reply:
I fixed it. However, in some other files like "widget/embedded/src/components/TokenList/TokenList.types.ts" it is used this way. Maybe we should correct them too.